app/src/main/res/values/strings.xml

```diff
@@ -49,11 +49,11 @@
     <string name="onboarding_finish_text2">You\'ve successfully completed the onboarding! You\'re now ready o boost your produtivity to a whole new level!</string>
     <string name="onboarding_finish_text3">We hope you enjoy and make good use of Kairos!</string>
 
-    <string name="onboarding_location_description">Kairos needs to know your work and home location so it can better define your context depending on your location.</string>
+    <string name="onboarding_location_description">Kairos needs to know your work and leisure location so it can better define your context depending on your location.</string>
```

Comment:
here is better to request home location instead of leisure location. You are asking the user's home (address) location, not leisure location